tests/test_modeling_tf_mt5.py

```diff
@@ -22,7 +22,17 @@
 if is_tf_available():
     import tensorflow as tf
 
-    from transformers import AutoTokenizer, TFAutoModelForSeq2SeqLM
+    from transformers import AutoTokenizer, T5Tokenizer, TFAutoModelForSeq2SeqLM, TFMT5ForConditionalGeneration
+
+
+@require_tf
+class TFMT5ModelTest(unittest.TestCase):  # no mixin with common tests -> most cases are already covered in the TF T5
+    @slow
+    def test_resize_embeddings(self):
+        model = TFMT5ForConditionalGeneration.from_pretrained("google/mt5-small")
+        tokenizer = T5Tokenizer.from_pretrained("google/mt5-small")
+        tokenizer.add_special_tokens({"bos_token": "", "eos_token": ""})
+        model._resize_token_embeddings(len(tokenizer))
```

Comment:
Can we make this test a bit more aggressive. There is no assert statement here. Could we check, *e.g.* that `len(tokenizer)` matches the model?

Reply:
You are right, there was no check with respect to whether the resizing was actually happening. Added more asserts that confirm that it is the case 👍